services/thermal_mgr/thermal_mgr.c

```diff
@@ -44,17 +45,60 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
 
+  if (thermalMgrQueueHandle == NULL || event == NULL) {
```

Comment:
- [ ] return invalid state if thermal mgr queue handle is null
- [ ] return invalid argument if event is null